phpBB/phpbb/console/application.php

```diff
@@ -35,7 +35,7 @@ class application extends \Symfony\Component\Console\Application
 	* @param string			$version	The version of the application
 	* @param \phpbb\user	$user		The user which runs the application (used for translation)
 	*/
-	public function __construct($name, $version, \phpbb\user $user)
+	public function __construct($name, $version, $user)
```

Comment:
the user is only used for the lang() function and in the installer we can't inject the user.

Anyway we need to change the constructor to `public function __construct($name, $version, \phpbb\language\language $language)`